content/_index.md

```diff
@@ -3,4 +3,18 @@ title: Introduction
 type: docs
 ---
 
-# Hello world
+# Welcome to the Research Engineering Group Handbook
+
+## What is this?
+
+This Handbook is written by and for [The Alan Turing Institute's](https://www.turing.ac.uk/) Research Engineering Group (REG). It is our attempt to describe both how we do work and how we aspire to. It is inspired by other openly available handbooks, such has [GitLab's All Remote](https://about.gitlab.com/company/culture/all-remote/guide/), [Basecamp's books](https://37signals.com/books) and (closer to home) [The Turing Way](https://the-turing-way.netlify.app/collaboration/remote-collab.html).
+
+## Who is the audience?
+
+Anyone! Most commonly it is used by members of REG in their day-to-day work. It also aims to be a helpful introduction to REG for anyone who might work with us, either as collaborators or - when the opportunity arises - as [future colleagues](https://www.turing.ac.uk/opportunities-turing/jobs).
```

Comment:
```suggestion
Anyone! Most commonly it is used by members of REG in their day-to-day work. It also aims to be a helpful introduction to REG for anyone who might work with us, either as collaborators or as [future colleagues](https://www.turing.ac.uk/opportunities-turing/jobs).
```

Or, if we think we should keep that, use '—' instead of a hyphen '-'.